docs/source/index.rst

```diff
@@ -1,26 +1,37 @@
 jupyter-repo2docker
 ===================
 
-**jupyter-repo2docker** is a tool to build, run, and push Docker
-images from source code repositories. repo2docker fetches a repository
+``jupyter-repo2docker`` is a tool to **build, run, and push Docker
+images from source code repositories** that run via a Jupyter server.
+
+``repo2docker`` fetches a repository
 (e.g., from GitHub or other locations) and builds a container image
 based on the configuration files found in the repository. It can be
 used to explore a repository locally by building and executing the
-constructed image of the repository.
+constructed image of the repository, or as a means of building images that
+are pushed to a Docker registry.
```

Comment:
I think we discussed this previously and @yuvipanda had a strong opinion that we do not state that repo2docker is a tool for building images that you then distribute. Let's double check both current thinking and feelings.

Reply:
I think using repo2docker as a tool for you to distribute is totally fine. I have a strong opinion that repo2docker is not a tool for producing **Dockerfiles** that you distribute.